src/java/org/apache/cassandra/utils/concurrent/AbstractFuture.java

```diff
@@ -494,11 +494,11 @@ public Future<V> awaitThrowUncheckedOnInterrupt() throws UncheckedInterruptedExc
     public String toString()
     {
         String description = description();
-        String state = state();
+        String state = mystate();
```

Comment:
why this change?

Reply:
Conflict with same name but different signature (pubic State state()) method in Future.